Sorting/RadixSort.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+    Radix sort
+    ----------
+    An efficient, key-index-based, comparision based sorting algorithm.
+  	It sorts based on the keys obtained on each index of the numbers to be sorteed.
+
+    Time complexity
+    ---------------
+    O(D * N), where N is the number of keys, D is the number of digits in input integers.
+
+    Space complexity
+    ----------------
+    O(N), where N is the number of elements and B is the base for representing the numbers(10 in case of decimal system)
+    This space is used for count sorting each key and temporarily creating a sorted array.
+*/
+
+#include <iostream>
+#include <vector>
+#include "SortingUtils.hpp"
+
+using namespace std;
+
+//No. of elements to be sorted
+size_t SIZE;
+
+//order is set to 1 if the sorting is done in ascending order, -1 if the sorting is done in descending order
+int order;
+
+//These factors depend on whether the sorting is needed to be done in ascending or descending order
+int mult_factor = 1;
+int add_factor = 0;
```

Comment:
Please don't make all variables global. Instead, declare them in the function where they're required and pass them to other functions when needed.
You should make `size`, `order`, `mult_factor` and `add_factor` local to `main()`.